src/main/resources/org/jenkinsci/plugins/scriptsecurity/sandbox/whitelists/generic-whitelist

```diff
@@ -1114,6 +1117,9 @@ staticMethod org.codehaus.groovy.runtime.StringGroovyMethods replaceFirst java.l
 staticMethod org.codehaus.groovy.runtime.StringGroovyMethods reverse java.lang.String
 staticMethod org.codehaus.groovy.runtime.StringGroovyMethods takeWhile groovy.lang.GString groovy.lang.Closure
 staticMethod org.codehaus.groovy.runtime.StringGroovyMethods takeWhile java.lang.CharSequence groovy.lang.Closure
+staticMethod org.codehaus.groovy.runtime.StringGroovyMethods toBoolean java.lang.Boolean
+staticMethod org.codehaus.groovy.runtime.StringGroovyMethods toDouble java.lang.Double
+staticMethod org.codehaus.groovy.runtime.StringGroovyMethods toInteger java.lang.Integer
```

Comment:
This isn't a real method, did you mean `StringGroovyMethods.toInteger(String)` and the nondeprecated version `StringGroovyMethods.toInteger(CharSequence)`?

Reply:
Oops, that's what I get for manually typing it out!